index.js

```diff
@@ -126,7 +167,10 @@ function myMap(array, callback) {
 // // => ["Hi, Ian!", "Hi, Chase!"]
 // console.log("%c----------", "color: red") 
 
-
+function greet(ele) {
+  greeting = `Hi, ${ele}!`;
```

Comment:
Be careful on line 171 here, if you declare a variable without `let` or `const` it will become a global variable (even if you declare it inside a function).